### PR TITLE
i18n(fr): WordPress guide

### LINKED ISSUE #4048: I18n(fr) : WordPress
I assign myself to translate WordPress file in French language. 

https://github.com/withastro/docs/blob/main/src/content/docs/en/guides/cms/wordpress.mdx

### PR DESCRIPTION
**Pull request by thomasbnt**
#### What kind of changes does this PR include?

- Translated content

#### Description

Close #4048

<!--

### CODE REVIEW

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
This should fix the broken link error logged.

**Comment by thomasbnt** [comment]
> This should fix the broken link error logged.

Yes, thanks!

**Comment by dreyfus92** [comment]
Btw, i also noticed that the file should go in this path `src/content/docs/fr/guides/cms/wordpress.mdx`

**Comment by thomasbnt** [comment]
> Btw, I also noticed that the file should go in this path `src/content/docs/fr/guides/cms/wordpress.mdx`

Owwwwwwwwwwwwwwww that is a big mistake 😵

File: src/content/docs/fr/guides/cms/wordpress.mdx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+Pour utiliser les [Custom Post Types (CPT)](https://learn.wordpress.org/lesson-plan/custom-post-types/) dans votre API WordPress (pas seulement `post` et `page`), vous devrez [les configurer dans votre tableau de bord WordPress](https://stackoverflow.com/questions/48536646/how-can-i-get-data-from-custom-post-type-using-wp-rest-api) ou [ajouter le support de l'API REST pour les types de contenu personnalisés](https://developer.wordpress.org/rest-api/extending-the-rest-api/adding-rest-api-support-for-custom-content-types/) dans WordPress.
+
+Cet exemple récupère les données d'un site WordPress dont les types de contenu ont déjà été configurés et exposés à l'API REST.
+:::
+
+### Affichage d'une liste d'articles WordPress
+
+La page `src/pages/index.astro` répertorie chaque dinosaure, avec une description et un lien vers sa propre page.
+
+<FileTree title="Project Structure">
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/cms/wordpress.mdx:72]:
```suggestion
<FileTree title="Structure de Projet">
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Je vous laisse une petite suggestion 😁

File: src/content/docs/fr/guides/cms/wordpress.mdx (hunk 1 above)
Comment (thomasbnt) [reply src/content/docs/fr/guides/cms/wordpress.mdx]: Merci!

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work peeps 🚀

**Pull request merged**